pubkey/pubkeyd.go

```diff
@@ -26,22 +26,21 @@ import (
 	"sync"
 	"time"
 
-	"github.com/pkg/errors"
-
 	"github.com/kpango/gache"
 	"github.com/kpango/glg"
+	"github.com/pkg/errors"
 	authcore "github.com/yahoo/athenz/libs/go/zmssvctoken"
 	"golang.org/x/sync/errgroup"
 )
 
-// Pubkeyd represent the daemon to retrieve public key data.
-type Pubkeyd interface {
-	StartPubkeyUpdater(ctx context.Context) <-chan error
-	UpdatePubkey(context.Context) error
+// Daemon represent the daemon to retrieve public key data.
+type Daemon interface {
```

Comment:
also change the file name?
`pubkeyd.go` -> `daemon.go`

Reply:
LGTM, @kevindiu  can you fix it?